prereise/gather/griddata/transmission/geometry.py

```diff
@@ -32,7 +33,7 @@ class Conductor(DataclassWithValidation):
     resistance_per_km: float = None
     gmr: float = None
     area: float = None
-    permeability: float = field(init=False)
+    permeability: float = field(init=False, default=None)
```

Comment:
I think simply do `permeability: float = None` is equivalent here? I might be wrong.

Reply:
I think you're right! Done.